ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/console/DefaultOutputCustomizer.java

```diff
@@ -185,32 +168,73 @@ private boolean endsWith(Path path, Path relativePath) {
         return true;
     }
 
-    /*
-     * Selects and shows the specified line of text in editor
+    /**
+     * Finds a file by its path, opens it in editor and sets the text selection and
+     * reveals according to the specified line and column numbers
+     * 
+     * @param file
+     * @param lineNumber
+     * @param columnNumber
      */
-    private void selectRange(EditorPartPresenter editor, int line) {
-        if (editor instanceof TextEditor) {
-            TextPosition startPosition = new TextPosition(line - 1, 0);
-            int lineOffsetStart = ((TextEditor) editor).getDocument().getLineStart(line - 1);
-            if (lineOffsetStart == -1) {
-                lineOffsetStart = 0;
-            }
+    static void openFileInEditorAndReveal(AppContext appContext, EditorAgent editorAgent, Path file,
```

Comment:
@benoitf  I have the same feeling. `Default*` was like a first impression (and because java is most used language, imho) - so, having two more customizers [at the moment] it doesn't look like a proper name for this kind of customizer. 
Shell I turn DefaultOutputCustomizer into an abstract class (with few utility methods, like `openFileInEditorAndReveal(...)`, `selectRange(...)` and set of abstract methods of `OutputCustomizer` interface) and a class like JavaOutputCustomizer (with realisazion for Java/JavaScript stacktraces)?